NEWS.md

```diff
@@ -1,4 +1,41 @@
 # NEWS
+
+v0.2.13
+-------
+
+## Features
+
+### Support for `@lazy`
+
+Starting version `0.2.13`, `ClimaDiagnostics` supports diagnostic variables
+specified with un-evaluated expressions (as provided by
+[LazyBroadcast.jl](https://github.com/CliMA/LazyBroadcast.jl)).
+
+Instead of
+```julia
+function compute_ta!(out, state, cache, time)
+    if isnothing(out)
+        return state.ta .- 273.15
+    else
+        out .= state.ta .- 273.15
+    end
+end
+```
+You can now write
+```julia
+import LazyBroadcast: @lazy
```

Comment:
I think we should probably be consistent, and stick to using `LazyBroadcast.lazy` instead of `LazyBroadcast.@lazy`.

On the one hand, it's more parenthesis, which is unfortunate, but on the other hand, there are far fewer things that can happen in function calls than things that can happen in macros, and we simply don't need a macro to get this behavior.

@dennisYatunin feels pretty strongly about it, I'm a bit on the fence, but ultimately I do agree that it's simpler

Reply:
Changed to `lazy`